submissions/DmytryjK/memory-pair-game/js/script.js

```diff
@@ -75,14 +75,17 @@ function onClickUserActions(e) {
   }
 
   if (activeCards.length >= 2) {
+    
+    currentListCards.forEach(card => {
+      card.classList.add('pointer-events');
```

Comment:
Class `pointer-events` does not describe his functionality please avoid this naming in the future.